docs/installLiberty.md

```diff
@@ -3,18 +3,18 @@
 The `installLiberty` task is used to download and install WebSphere Liberty server. The task can also upgrade your Liberty runtime from an ILAN to an IPLA license with a license JAR file [setup](#installing-your-upgrade-license) and [license configuration](#license-configuration).  
 
 The task can download the WebSphere Liberty runtime archive in three ways: 
-* from a specified location (via `runtimeUrl`)
+* from a specified location using `runtimeUrl`
 * from the [Liberty repository](https://developer.ibm.com/wasdev/downloads/) based on a version and a runtime type
-* from the [Maven Central repository](http://search.maven.org/) using the dependencies block.  
+* from the [Central repository](https://public.dhe.ibm.com/ibmdl/export/pub/software/websphere/wasdev/downloads/wlp/index.yml) using the `libertyRuntime` dependencies configuration.  
```

Comment:
When I look at maven central, they tend to spell it `The Central Repostory`.

That link doesn't look right - not the right link for maven central.  The link to index.yml goes on "index.yml" in the next paragraph.  Sorry if my comment didn't make sense.  There's not a + to add comments on that sentence.